### DISCUSSION TITLE
Transaction confirmation on quorum Blockchain network

### DISCUSSION THREAD

**Discussion by tkb2501** [ConsenSys/quorum#1649] [General]
Hi Team,

We are working on a production application based on private quorum Blockchain network(using QBFT), and we are facing issue for transaction confirmation as transaction id is created initially before transaction confirmation and the Blockchain transaction output is not the processed output. So is there a way so that we can identify if the transaction is processed successfully?

**Comment by baptiste-b-pegasys** [comment]
With the transaction receipt, you know it's been processed with or without reversion.
